docs/source/deployment/airflow.md

```diff
@@ -166,6 +166,69 @@ astro deploy
 
 ![](../meta/images/astronomer_cloud_deployment.png)
 
+## How to run a Kedro pipeline on Amazon AWS Managed Workflows for Apache Airflow (MWAA)
+
+### Kedro project preparation
+MWAA, or Managed Workflows for Apache Airflow, is an AWS service that makes it easier to set up, operate, and scale Apache Airflow in the cloud. Deploying a Kedro pipeline to MWAA is similar to Astronomer, but there are some key differences: you need to store your project data in an AWS S3 bucket and make necessary changes to your Data Catalog. Additionally, you must configure how you upload your Kedro configuration, install your Kedro package, and set up the necessary environment variables.
+1. Complete steps 1-4 from the [Create, prepare and package example Kedro project](#create-prepare-and-package-example-kedro-project) section.
+2. Your project's data should not reside in the working directory of the Airflow container. Instead, [create an S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/creating-bucket.html) and [upload your data folder from the new-kedro-project folder to your S3 bucket](https://docs.aws.amazon.com/AmazonS3/latest/userguide/upload-objects.html).
+3. Modify the Data Catalog to reference data in your S3 bucket by updating the filepath and add credentials line for each Dataset in the `new-kedro-project/conf/airflow/catalog.yml`. Add the S3 prefix to the filepath as shown below:
+```shell
+companies:
+  type: pandas.CSVDataset
+  filepath: s3://your_S3_bucket/data/01_raw/companies.csv
+  credentials: dev_s3
+```
+4. [Set up AWS credentials](https://docs.aws.amazon.com/keyspaces/latest/devguide/access.credentials.html) to provide read and write access to your S3 bucket. Update `new-kedro-project/conf/local/credentials.yml` with your AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY:
+```shell
+dev_s3:
+  client_kwargs:
+    aws_access_key_id: *********************
+    aws_secret_access_key: ******************************************
+```
```

Comment:
I haven't tried this yet, but do the credentials in the `local/credentials.yml` work on the cloud platform? This file ideally shouldn't be uploaded to the cloud at all. Maybe setting up env variables?

Reply:
I tried that approach with the `local` environment, and it works. We can change the environment from `local` to `prod`, but am I correct in understanding that there isn't a big difference? 

For setting environment variables, I am using an `env_var.sh` file that is located in the same S3 bucket as the `conf` folder, which is currently packed into `plugins.zip`. Therefore, I don't think it makes much sense to use environment variables for security reasons because, ultimately, they will be stored in the same S3 bucket. However, this S3 bucket is secure and does not have public access, so I think it's not a problem.